README.md

```diff
@@ -2091,6 +2091,37 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='remove-blank-lines-between-chained-functions'></a>(<a href='#remove-blank-lines-between-chained-functions'>link</a>) **Remove blank lines between chained functions.** [![SwiftFormat: blanklinesbetweenchainedfunctions](https://img.shields.io/badge/SwiftFormat-blankLinesBetweenChainedFunctions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#blanklinesbetweenchainedfunctions)
+
+  <details>
+
+  #### Why?
+
+  Improves readability and maintainability, making it easier to see the sequence of functions that are applied to the object.
+
+  ```swift
+  // WRONG
+  let planets = ["Mercury", "Venus", "Earth", "Mars"]
+
+  var transformedPlanetNames: [String] {
+      planets
+        .map { $0.uppercased() } // Making names uppercase
+        
+        .map { $0 + "!" } // Adding excitement with an exclamation mark
```

Comment:
Noticed there's too much indentation here

```suggestion
    planets
      .map { $0.uppercased() } // Making names uppercase
        
      .map { $0 + "!" } // Adding excitement with an exclamation mark
```

Reply:
Good catch - I notice the 4 spaces, but assumed that it was the preference for this repo. Should have checked the other code examples, where it's clearly not the case. I'll update it.